src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -413,16 +414,16 @@ transactionSign (
             return e;
     }
 
-    if (!tx_json.isMember ("Sequence"))
-        tx_json["Sequence"] = ledgerFacade.getSeq ();
-
     if (!tx_json.isMember ("Flags"))
         tx_json["Flags"] = tfFullyCanonicalSig;
 
     if (verify)
     {
+        if (!tx_json.isMember ("Sequence"))
```

Comment:
I think we've pretty much generally agreed to use them everywhere we have static names of object members. The primary benefit they provide is when you use them in an object -- they avoid an allocate/copy/free for the name of the object member when it's created and every time it's copied.